PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/RootLayoutController.java

```diff
@@ -79,13 +62,35 @@
    */
   @Override
   public void initialize(URL location, ResourceBundle resources) {
-    heatmapManager = new HeatmapManager(heatmapPane);
     initializeSelectionManager();
-    initializeTreeIcon();
-    initializeGraphIcon();
+    treeManager = TreeManager.loadView(selectionManager);
+    drawGraphs = DrawComparedGraphs.loadView();
+    mainPane.getItems().add(treeManager.getTreePane());
+    mainPane.getItems().add(drawGraphs.getGraphPane());
     graph = new GfaReader("TB10.gfa").read();
     mainGraphOrder = new GraphOrdererThread(graph);
     mainGraphOrder.start();
+    loadTree();
+  }
+
+  /**
+   * Load the data into the root layout.
+   *
+   * @param controller the controller of the root layout.
```

Comment:
Yes, the file path is hardcoded because there is no file selection option yet. When this feature is implemented the `loadTree()` method can easily be adjusted though.